public/app/features/alerting/unified/RuleList.test.tsx

```diff
@@ -187,6 +191,11 @@ const configureMockServer = () => {
     message: 'rule group updated successfully',
     updated: ['foo', 'bar', 'baz'],
   });
+  mockAlertRuleApi(server).rulerRuleGroup(GRAFANA_RULES_SOURCE_NAME, 'NAMESPACE_UID', 'groupPaused', {
+    name: 'group-1',
+    interval: '1m',
+    rules: [mockRulerGrafanaRule()],
```

Comment:
It would be even nicer if the mocked response had more than one rule in it, so we could check that additional ones were fetched - but I think its not going to result in a particularly nice test anyway, as we'd have to check the server was called with certain payload which we're trying to avoid 🤷 So this is fine for now I'd say :+1: